package.json

```diff
@@ -47,7 +47,7 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.0",
     "typescript": "^4.3.5",
-    "web-components": "department-of-veterans-affairs/component-library#wc-v0.9.0"
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-bindings-packaging"
```

Comment:
We'll want to make sure these component changes get released before this `formulate` PR gets merged.